stage_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Get names of all cronjobs, if not found any ignore the error message
+# to not let k8s raise an error with an exit code indicating success
+CRONJOBS=$(oc get cronjobs 2>/dev/null | awk 'NR>1 {print $1}')
```

Comment:
I'm getting 
```
ERROR: no cronjobs found
```
although, CronJob(s) exist.

Please don't redirect errors to `/dev/null`.
```
oc: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by oc)
oc: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by oc)
oc: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by oc)
```

Reply:
Use the client for RHEL 8:
https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/openshift-client-linux-amd64-rhel8.tar.gz

Reply:
I redirected errors because I was concerned that when `kubectl` [doesn't find a resource and returns success](https://github.com/kubernetes/kubectl/issues/821) that it would equate to a successful job run.  

My bad, thank you for catching this.